Modules/EMCAL/src/RawErrorCheckAll.cxx

```diff
@@ -0,0 +1,129 @@
+#include "QualityControl/MonitorObject.h"
+#include "EMCAL/RawErrorCheckAll.h"
+#include "QualityControl/QcInfoLogger.h"
+#include "QualityControl/Quality.h"
+
+// ROOT
+#include <TH1.h>
+#include <TH2.h>
+#include <TPaveText.h>
+#include <TLatex.h>
+#include <TList.h>
+#include <TRobustEstimator.h>
+#include <ROOT/TSeq.hxx>
+#include <iostream>
+#include <vector>
+#include <TCanvas.h>
+#include <string>
+#include <TGraph.h>
+#include <queue>
+
+using namespace std;
```

Comment:
`Using` of whole namespaces must never be applied except the one declared in the header. In particular don't apply `using namespace std`, or of any member of the standard namespace.